libs/common/src/sql-db/entity/utilisation-report.ts

```diff
@@ -0,0 +1,29 @@
+import { Column, Entity, PrimaryGeneratedColumn } from 'typeorm';
+import { UtilisationReportReconciliationStatus } from '../../types';
+import { ReportPeriod } from './partials';
+
+@Entity('UtilisationReport')
```

Comment:
If we don't enter a value here (i.e.  just `@Entity()` instead if `@Entity('UtilisationReport')`) then the table name will be based off of the class name. I wanted to add `...Entity` as a suffix to the class name though so it's clear what this is when imported into other packages.

This casing of the table name (UpperCamelCase), and the fact that it's singular `UtilisationReport` instead of `UtilisationReports` is based off of the conventions that Microsoft follow themselves (see examples [here](https://github.com/microsoft/sql-server-samples)). This doesn't seem like a required standard though, but instead more of a preference. If we hadn't specified the table name explicitly then TypeORM would have actually named the table with snake_case `utilisation_report_entity`.

I note that Microsoft also uses UpperCamelCase for property names too, but I haven't overridden those (just because I wasn't forced to). Maybe we want to be consistent. I'm guessing use of UpperCamelCase by Microsoft is only used to be consistent with C# conventions, so maybe we don't need to follow that here.

What do we think?